### PR TITLE
Removed updating functionality from the Loading Screen window

### PR DESCRIPTION
**Pull request by jamcohen**
- Simplified the Loading Screen window by removing the buttons associated with the AssetBundle URL.
- Removed the dialog asking "Would you like to replace any existing Scenes in Build with the loading screen scene?". Now instead of removing scenes in build settings, we just disable them.
- Removed the engine config and added a save dialog after uses hit "Create Loading Scene..."
- The Loading Scene window now closes after the user saves the scene. Then we automatically select the LoadingScreen component in the Hierarchy.

### CODE REVIEW

File: GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs [hunk 1]
```diff
@@ -81,36 +83,42 @@ public static void GenerateScene(string assetBundleUrl, Texture2D loadingScreenI
...
+            }
+            else
+            {
+                EditorBuildSettings.scenes = buildScenes;
+            }
         }
 
-        // Visible for testing
-        internal static void SetMainSceneInBuild(string pathToScene)
+        private static void FindSceneInBuildSettings()
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs]: Is this empty now? Delete?

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs [hunk 2]
```diff
@@ -262,106 +259,60 @@ private void OnGuiLoadingScreenSelect()
...
-            EditorGUILayout.Space();
-
-            EditorGUILayout.EndVertical();
-            EditorGUILayout.Space();
-
-            EditorGUILayout.Space();
-            EditorGUILayout.LabelField("Select Loading Screen Image", EditorStyles.boldLabel);
-            EditorGUILayout.BeginVertical(UserInputGuiStyle);
-            EditorGUILayout.Space();
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider); // Draws a horizontal line
```
Comment (thanwah) [inline GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: This doesn't really look like other UI that we have. Should we instead do separate shaded sections? Or just combine them outright?

File: GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs (hunk 1 above)
Comment (jamcohen) [reply GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs]: Resolved, thanks for catching that.

File: GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs (hunk 2 above)
Comment (jamcohen) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: I'll try both and we can see which we prefer.

Comment (jamcohen) [reply GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs]: Resolved

**Pull request merged**